src/base/http/types.h

```diff
@@ -64,8 +71,10 @@ namespace Http
     const char CONTENT_TYPE_PNG[] = "image/png";
     const char CONTENT_TYPE_TXT[] = "text/plain";
     const char CONTENT_TYPE_SVG[] = "image/svg+xml";
+    const char CONTENT_TYPE_FORM_ENCODED[] = "application/x-www-form-urlencoded";
+    const char CONTENT_TYPE_FORM_DATA[] = "multipart/form-data";
 
-    // portability: "\r\n" doesn't guarantee mapping to the correct value
+    // portability: "\r\n" doesn't guarantee mapping to the correct symbol
```

Comment:
@Chocobo1, as I remember, you introduced this statement earlier. Where can I get the source of this info?

Reply:
https://en.wikipedia.org/wiki/Newline#In_programming_languages